services/transcode/audio_test.go

```diff
@@ -27,3 +27,8 @@ func Test_AudioSplit(t *testing.T) {
 
 	spew.Dump(files)
 }
+
+func Test_AudioSilence(t *testing.T) {
```

Comment:
Would it make sense to add a test file into the project. Obviously not a few gig one.. :)